pkgs/tools/admin/pulumi/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchurl, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "pulumi-${version}";
+  version = "0.16.2";
+
+  src = if (stdenv.system == "x86_64-linux")
```

Comment:
`stdenv.hostPlatform.system` should work better in cross-compilation. Also multiple `if` statements can be nicely refactored as lookup in attrset (can look how it's done in `firefox-bin` or `oraclejdk`).

Reply:
thanks! great feedback!